### DISCUSSION TITLE
Bulk Labeling Selections delete feature?

### DISCUSSION THREAD

**Discussion by eringiglio** [talmolab/sleap#1249] [Ideas]
Hi! Great program, really enjoying it. Is there any plan to incorporate a bulk selection feature for Labeling Suggestions to make selectively deleting easier? I'm running a very large project with a lot of undergrad help, and it would be really helpful to me to be able to delete suggestions on videos I have inferenced or, when using the predicting score feature to identify unlabeled predictions, to be able to delete all the suggested frames that are already labeled. I think that would make it easier to show students where new frames needing fixing are.

Thanks so much for all the good work.

Erin

**Comment by roomrys** [comment]
Hi @eringiglio,

That is not something we have (other than the option to clear all suggestions), but it is a good usability feature! I'm thinking the standard `click + Shift + click` for sequential selection or `Ctrl + click + ... + click` for non-sequential selection. However, something we have which might be useful is sorting the Labeling Suggestions via the columns. Namely, if you select the "Labeled" column it will sort the table based on how many instances/animals are labeled in a frame.

As you have likely noticed, currently the Labeling Suggestions table also doubles as a (not-all-encompassing) Labels table to keep track of what you have already labeled. Deleting the suggestions will also delete your only visual for tracking what has been labeled. We really need another table to just keep track of all the labeled frames (design consideration: ensure the access pattern plays well with the Labels data structure - or vice versa). And then we should sculpt each table to include features that work for that tables functionality. Your idea above is a great general feature for any table!

Thanks for the idea! 😄 
Liezl
